data/resources/style.css

```diff
@@ -263,14 +265,20 @@ messagemedia overlay > messageindicators, messagesticker messageindicators {
 
 /* Placeholder */
 .message-entry > overlay > label {
-  margin: 0 9px;
+  margin-left: 9px;
```

Comment:
This doesn't work for RTL.

![image](https://user-images.githubusercontent.com/23530586/200792206-8b9ba418-3a90-4ebd-90a6-2ce470a15b04.png)

Reply:
Same thing for the emoji button.